### ISSUE TITLE
Pachctl debug dump fails 

### ISSUE THREAD

**Issue by JimmyWhitaker** [pachyderm/pachyderm#6329]
In Pachyderm 1.13 
`pachctl` hangs on `pachctl list job` in the Iris example

When running `pachctl debug dump`, it errors with: 
`Unknown service debug.Debug`

**Comment by msteffen** [comment]
In our on-call handoff, @actgardner inferred that `Unknown service debug.Debug` indicated a `pachctl` 2.0 binary trying to communicate with `pach` 1.13.x server.

Given that, I'm marking this issue closed for triage purposes, but if this diagnosis isn't correct please reopen

**Comment by JimmyWhitaker** [comment]
I don't believe that was the case in this instance. We were using IDE to perform the test, specifically so we would have the 1.13 binary performing the communication with the 1.13 server.
